rpcs3/Emu/NP/np_handler.cpp

```diff
@@ -149,8 +196,35 @@ void np_handler::init_NP(u32 poolsize, vm::ptr<void> poolptr)
 		break;
 	case np_psn_status::fake:
 	{
-		strcpy_trunc(online_name.data, "RPCS3's user");
-		strcpy_trunc(avatar_url.data, "https://i.imgur.com/AfWIyQP.jpg");
+		std::memcpy(online_name.data, "RPCS3's user", std::min(sizeof(online_name.data), std::string_view("RPCS3's user").size()));
+		std::memcpy(avatar_url.data, "https://i.imgur.com/AfWIyQP.jpg", std::min(sizeof(avatar_url.data), std::string_view("https://i.imgur.com/AfWIyQP.jpg").size()));
```

Comment:
Could this be hosted on rpcs3.net instead?

Reply:
This manual "min size" memcpy is bad, use strcpy_trunc.